Marlin/configuration_store.cpp

```diff
@@ -173,6 +180,9 @@ void Config_StoreSettings()  {
   char ver[4] = "000";
   int i = EEPROM_OFFSET;
   EEPROM_WRITE_VAR(i, ver); // invalidate data first
+
```

Comment:
Now i = 4 you must add i 2 for uint16 eeprom-checksum or write eeprom_checksum 0...